src/main/java/app/controller/TaskController.java

```diff
@@ -0,0 +1,73 @@
+package app.controller;
+
+import app.dto.task.CreateTaskRequestDto;
+import app.dto.task.TaskDtoWithoutLabelsAndComments;
+import app.dto.task.TaskResponseDto;
+import app.service.task.TaskService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@Tag(name = "Task management", description = "Endpoints for managing tasks")
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/api/projects/{projectId}/tasks")
```

Comment:
![image](https://github.com/jv-feb24-team04/task-management-app/assets/159646520/f2f374f0-53e9-4d76-9db3-45eaef14b54a)

Reply:
fix paths and related methods if needed

Reply:
But how would I get tasks for a project if there if no project ID in the path?

Reply:
I asked in chat earlier and me and Valik came to the conclusion that it was better to do it like that. If needed I will change this path later

Reply:
`/api/tasks/byproject/{projectId}` ? or something like that
with this path `/api/projects/{projectId}/tasks` you're requesting something from task-related logic but in fact, it's the path for `ProjectController`.
You will get the same path variable but the path hierarchy will be consistent